package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "contentstack-cli-content-type",
```

Comment:
@abhinav-from-contentstack The GH workflow is failing in this PR, can you please fix that?